src/main/java/org/wildfly/security/auth/provider/jdbc/QueryConfiguration.java

```diff
@@ -62,7 +67,19 @@ public DataSource getDataSource() {
      *
      * @return
      */
-    public List<ColumnMapper> getColumnMappers() {
-        return Collections.unmodifiableList(this.keys);
+    List<ColumnMapper> getColumnMappers() {
+        return Collections.unmodifiableList(this.columnMappers);
+    }
+
+    <T extends ColumnMapper> List<T> getColumnMappers(Class<T> mapperType) {
+        List<T> attributeMappers = new ArrayList<>();
+
+        for (ColumnMapper columnMapper : this.columnMappers) {
+            if (mapperType.isInstance(columnMapper)) {
+                attributeMappers.add((T) columnMapper);
```

Comment:
Here's another unsafe cast that can be changed to use ``mapperType.cast()`` instead.